### PR TITLE
Popup message updated

### PR DESCRIPTION
**Pull request by nitinkgp23**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def student_register(request):
                 slack_notification("Unable to send mail to the following student :\n{}".format(
                     form_dict))
             flag="True"
-            msg="You have been successfully registered."
+            msg="Don't forget to check your <a href='https://gmail.com' target='blank'>email</a> or the <a href='/resources' target='_blank'>Resources</a> page!"
```
Comment (AvijitGhosh82) [inline app.py]: What if someone is not using Gmail?

Comment (AvijitGhosh82) [inline app.py]: Why do we need to provide a href at all?

Comment (nitinkgp23) [inline app.py]: So that they check their email instantly ?
Using href will prompt them to click on it.

**Comment by OrkoHunter** [comment]
Closing due to https://github.com/kossiitkgp/kwoc/pull/57/commits/989df45fdbfb350a6a8f43bacdcdfd01effdde82

**Pull request closed without merging**